style.css

```diff
@@ -0,0 +1,718 @@
+  flex-direction: column;
+  align-items: center;
+}
+
+.about-logo-section-heading {
+  font-family: 'lato', sans-serif;
+  font-weight: 600;
+  font-size: 40px;
+  color: #272a31;
+}
+
+.about-logo-section-description {
+  text-align: center;
+  font-size: 18px;
+  font-family: 'lato', sans-serif;
+  font-weight: 600;
+  color: #272a31;
+}
+
+.about-section-logo-image {
+  border: #272a31 solid 2px;
+  width: 50%;
+}
+
+.our-features {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 8%;
+}
+
+.our-features-heading {
+  text-align: center;
+  font-family: 'lato', sans-serif;
+  color: #272a31;
+  font-size: 40px;
+  font-weight: 600;
+}
+
+.our-features-description {
+  text-align: center;
+  font-family: 'lato', sans-serif;
+  font-size: 18px;
+  color: #272a31;
+}
+
+.feature-programs {
+  width: 100%;
+}
+
+.feature-program-name {
+  margin: 0 3%;
+  background-image:
+    linear-gradient(
+      rgba(228, 43, 23, 0.5),
+      rgba(221, 43, 23, 0.5)
+    ),
+    url(./images/fruit-salad.jpg);
+  padding: 10% 20%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 5%;
+}
+
+.feature-program-name2 {
+  margin: 0 3%;
+  background-image:
+    linear-gradient(
+      rgba(228, 43, 23, 0.5),
+      rgba(221, 43, 23, 0.5)
+    ),
+    url(./images/nutrtion\ seminar.png);
+  padding: 10% 20%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 5%;
+}
+
+.feature-program-heading {
+  font-size: 20px;
+  font-weight: 900;
+  color: #f0f2f5;
+  text-align: center;
+  font-family: 'lato', sans-serif;
+}
+
+.feature-program-description {
+  font-weight: 900;
+  color: #f0f2f5;
+  text-align: center;
+}
+
+.project-rights-about {
+  background-color: #272a31;
+  display: flex;
+  align-items: center;
+  padding: 4%;
+}
+
+.footer-content {
+  color: #d3d3d3;
+}
+
+.footer-logo-about {
+  width: 30%;
+  height: 50%;
+  margin-right: 10%;
+}
+
+.footer-text {
+  font-family: "lato", sans-serif;
+  color: #272a31;
+  font-weight: 400;
+  margin-top: 20px;
+  font-size: 1.2rem;
+}
+
+@media only screen and (min-width: 769px) {
+  .featured-program-item-desktop {
+    background-color: rgba(88, 89, 90, 0.8);
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    margin-right: 0.5%;
+    width: 15%;
+    padding: 1%;
+  }
+
+  .nav-button-link {
+    border: #c21b09 solid 4px;
+    padding: 2%;
+  }
+
+  .nav-button {
+    color: #c21b09;
+  }
+
+  .mobile-hide {
+    display: block;
+  }
+
+  .mobile-works {
+    display: none;
+  }
+
+  .social-media-section {
+    background-color: #272a31;
+    align-items: center;
+  }
+
+  .social-media {
+    list-style: none;
+    align-items: center;
+    text-align: center;
+  }
+
+  .social-media-item {
+    text-decoration: none;
+    color: #d3d3d3;
+  }
+
+  .social-media-row {
+    align-items: center;
+    text-align: center;
+    padding-top: 0.8%;
+  }
```

Comment:
- [ ] Greeat work with the project :clap: , As I can see the top darker menu bar which contains the social links are a bit different from the original design guidelines, please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217483298-1af7c62b-d7dd-4ea9-9236-f3b6bb2f8d04.png)

how it is in the original design guideline :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217483722-1b4a5c6a-2218-4d95-8da9-dab52222e5cb.png)

notice the my page link is horizontal this will also fix the height which should be less